core/src/main/scala/cr/pulsar/Consumer.scala

```diff
@@ -29,10 +29,29 @@ import org.apache.pulsar.client.api.{
 import scala.util.control.NoStackTrace
 
 trait Consumer[F[_], E] {
+
+  /**
+    * Acknowledge for a single message.
+    */
   def ack(id: MessageId): F[Unit]
+
+  /**
+    * Negative acknowledge for a single message.
+    */
   def nack(id: MessageId): F[Unit]
+
+  /**
+    * It consumes [[Consumer.Message]]s, which contain the ID and the PAYLOAD.
+    *
+    * If you don't need manual [[ack]]ing, consider using [[autoSubscribe]] instead.
+    */
   def subscribe: Stream[F, Consumer.Message[E]]
 
+  /**
+    * Use together with `autoAck`.
+    */
+  def autoSubscribe: Stream[F, E]
```

Comment:
I was thinking about the same but combining it with autoAck by default. Is there a use case when you would use autoSubscribe without autoAck?

Reply:
Makes sense, will update :+1: